src/store/Selectors/GraphData.ts

```diff
@@ -55,9 +63,10 @@ const decorateGraphData = (graphData: any) => {
       workload: undefined
     }
   };
+  const decoratedGraph: DecoratedGraphElements = {};
   if (graphData) {
     if (graphData.nodes) {
```

Comment:
A couple of comments.  I think you need to cast `decoratedNode.data` and then remove the comment line. And we don't want to have a `traffic` field in the decorated node data.  I think this whole if-clause should look like:

```
if (graphData.nodes) {
      decoratedGraph.nodes = graphData.nodes.map((node: GraphNodeWrapper) => {
        const decoratedNode = { ...node };
        // parse out the traffic data into top level fields for the various protocols. This is done
        // to be back compatible with our existing ui code that expects the explicit http and tcp fields.
        // We can then set the 'traffic' field undefined because it is unused in the cy element handling.
        // TODO: refactor the code to use the traffic structure.
        if (decoratedNode.data.traffic) {
          const traffic = decoratedNode.data.traffic;
          decoratedNode.data.traffic = undefined;
          traffic.map(protocol => {
            decoratedNode.data = { ...protocol.rates, ...decoratedNode.data };
          });
        }
        decoratedNode.data = <DecoratedGraphNodeData> { ...elementsDefaults.nodes, ...decoratedNode.data };
        return <DecoratedGraphNodeWrapper> decoratedNode;
      });
    }
```